README.md

```diff
@@ -4,9 +4,11 @@ Demo to show a real-time trading dashboard for three different exchanges.
 
 The complete crypto-trading demo has two components:
 
-1. A stream application.
-2. A UI application written in Reactjs
+1. A backend built with Macrometa a Document Collection, multiple Streams, and a Stream Worker.
```

Comment:
Change this to an unordered list. Only use numbered lists when order matters.

Also, nothing in this sentence should be capitalized except for Macrometa.